src/Cqm/Qdm/Traits/PatientExtension.php

```diff
@@ -5,7 +5,7 @@
 trait PatientExtension
 {
     public $_fullName;
-    public $_openEmrPid;
+    public $_id;
 
```

Comment:
Was this meant to change

Reply:
This was me trying to preserve the PID through the CQM calculator to try to map results to openemr patients, but all extra fields are stripped.